xla/ffi/api/c_api.h

```diff
@@ -178,6 +178,11 @@ typedef enum {
   XLA_FFI_DataType_C64 = 15,
   XLA_FFI_DataType_C128 = 18,
   XLA_FFI_DataType_TOKEN = 17,
+  XLA_FFI_DataType_F8E5M2 = 19,
+  XLA_FFI_DataType_F8E4M3FN = 20,
+  XLA_FFI_DataType_F8E4M3B11FNUZ = 21,
+  XLA_FFI_DataType_F8E5M2FNUZ = 22,
+  XLA_FFI_DataType_F8E4M3FNUZ = 23,
```

Comment:
See https://github.com/openxla/xla/blob/main/xla/xla_data.proto#L74-L102 - enum values must be consistent